.github/workflows/build_image.yaml

```diff
@@ -0,0 +1,25 @@
+name: Gradle try build
+
+on:
+  push:
+    tags:
+      - 'v*'
```

Comment:
I've checked, [it's using glob pattern](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#:~:text=The%20branches%2C%20branches%2Dignore%2C%20tags%2C%20and%20tags%2Dignore%20keywords%20accept%20glob%20patterns)